_distro_map.yml

```diff
@@ -170,7 +173,7 @@ openshift-dedicated:
     enterprise-3.11:
       name: '3'
       dir: dedicated/3
-    enterprise-4.17:
+    enterprise-4.18:
```

Comment:
@karanthshashank, will you confirm that OSD and ROSA will start tracking 4.18 at OCP 4.18 GA?